crates/ironclaw_gateway/static/styles/primitives/badges.css

```
@@ -0,0 +1,170 @@
+}
+
+.badge.in_progress::before {
+  content: '';
+  position: absolute;
+  left: 6px;
+  top: 50%;
+  transform: translateY(-50%);
+  width: 6px;
+  height: 6px;
+  border-radius: var(--v2-radius-full);
+  background: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)